tests/test_auth.py

```diff
@@ -0,0 +1,37 @@
+import pgautofailover_utils as pgautofailover
+from nose.tools import *
+import time
+
+cluster = None
+node1 = None
+node2 = None
+
+def setup_module():
+    global cluster
+    cluster = pgautofailover.Cluster()
+
+def teardown_module():
+    cluster.destroy()
+
+def test_000_create_monitor():
+    monitor = cluster.create_monitor("/tmp/monitor", authMethod="md5")
+    monitor.wait_until_pg_is_running()
+
+def test_001_init_primary():
+    global node1
+    node1 = cluster.create_datanode("/tmp/node1", authMethod="md5")
```

Comment:
I think we should call the method that sets the passwords at this point in the test file, after having created a cluster with MD5 authentication policy.

Reply:
added set_user_password method to set individual user passwords